packages/ag-charts-website/e2e/keyboard-nav.spec.ts

```diff
@@ -98,4 +98,68 @@ test.describe('keyboard-nav', () => {
             });
         });
     }
+
+    test('polar chart', async ({ page }) => {
+        const { url } = toExamplePageUrl('pie-series', 'simple-pie', 'vanilla');
+
+        await gotoExample(page, url);
+
+        await page.locator(SELECTORS.canvasCenter).first().click();
+
+        for (let datum = 0; datum < 5; datum += 1) {
+            if (datum === 0) {
+                await page.keyboard.press('ArrowLeft');
+            } else {
+                await page.keyboard.press('ArrowRight');
+            }
+            await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`pie-${datum + 1}-highlight.png`);
+        }
+    });
+
+    test('topology chart', async ({ page }) => {
+        const { url } = toExamplePageUrl('map-shapes', 'multiple-series', 'vanilla');
+
+        await gotoExample(page, url);
+
+        await page.locator(SELECTORS.canvasCenter).first().click();
+
+        for (let datum = 0; datum < 5; datum += 1) {
+            if (datum === 0) {
+                await page.keyboard.press('ArrowLeft');
+            } else {
+                await page.keyboard.press('ArrowRight');
+            }
+            await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`map-shape-${datum + 1}-highlight.png`);
+        }
```

Comment:
ditto: **DAMP** instead of **DRY**.
```suggestion
        await page.keyboard.press('ArrowLeft');
        await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`map-shape-1-highlight.png`);
        await page.keyboard.press('ArrowRight');
        await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`map-shape-2-highlight.png`);
        await page.keyboard.press('ArrowRight');
        await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`map-shape-3-highlight.png`);
        await page.keyboard.press('ArrowRight');
        await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`map-shape-4-highlight.png`);
        await page.keyboard.press('ArrowRight');
        await expect(page.locator(SELECTORS.canvasCenter)).toHaveScreenshot(`map-shape- 5-highlight.png`);
```